Plugins/explicitDependencyImportCheckPlugin/ExplicitDependencyImportCheckPlugin.swift

```diff
@@ -25,20 +37,40 @@ extension ExplicitDependencyImportCheckPlugin {
         transitiveDeps: Set<String>,
         forTarget target: Target,
         inContext context: PluginContext
-    ) throws -> Command {
-        var transitiveDependenciesErrorText = "error: \(transitiveDeps.count) Transitive dependencies found for \(target.name) 🚨🚨🚨"
-        for transitiveDep in transitiveDeps {
-            transitiveDependenciesErrorText += "\n"
-            transitiveDependenciesErrorText += "👉 \(transitiveDep) is a transitive dependency"
-        }
+    ) throws -> [Command] {
+        guard !transitiveDeps.isEmpty else { return [] }
+
+        var transitiveDependenciesErrorText = "error: \(transitiveDeps.count) Transitive dependencies found for \(target.name) 🚨🚨🚨\n"
+        transitiveDependenciesErrorText += transitiveDeps.map { "👉 \($0) is a transitive dependency" }
+                                                         .joined(separator: "\n")
+
         let echoTool = try context.tool(named: "echo")
-        return .buildCommand(
+        return [.buildCommand(
             displayName: "Transitive Dependencies Report",
             executable: echoTool.path,
             arguments: [transitiveDependenciesErrorText],
             environment: [:]
-       )
-        
+       )]
+    }
+
+    func exportUnusedDependenciesAsWarnings(
+        unusedDependencies: Set<String>,
+        forTarget target: Target,
+        inContext context: PluginContext
+    ) throws -> [Command] {
+        guard !unusedDependencies.isEmpty else { return [] }
+
+        var warningText = "warning: \(unusedDependencies.count) Extraneous dependencies found for \(target.name) ⚠️⚠️\n"
+        warningText += unusedDependencies.map { "👉 \($0) is an extraneous depdnency" }
```

Comment:
Looks like there is a typo here `depdnency `

Reply:
Looks like there is a typo here depdnency